lib/internal/Magento/Framework/Module/Setup/Migration.php

```diff
@@ -694,10 +704,12 @@ public function getCompositeModules()
      *
      * @param string $encodedValue
      * @param int $objectDecodeType
-     * @return mixed
+     * @return array|bool|float|int|mixed|null|string
```

Comment:
Again not happy that the return and the throws have changed been added but not really sure if this is ok/if there is a better option in this case.